pkg/types/spec/validations.go

```diff
@@ -606,14 +606,18 @@ func multiModelValidation(fieldName string) *cr.StructFieldValidation {
 				{
 					StructField: "Path",
 					StringPtrValidation: &cr.StringPtrValidation{
-						Required: false,
+						Required:        false,
+						InvalidPrefixes: []string{".", "./", "/"},
```

Comment:
Is it necessary to have this check, since it would get caught on the line below (and that seems like it might be a nicer error message in that case)?

Reply:
No, it isn't required. Just thought it would be nice. What do you think?